src/Admin/AdminInterface.php

```diff
@@ -659,14 +650,28 @@ public function getListMode();
      */
     // public function isCurrentRoute(string $name, ?string $adminCode = null): bool;
 
+    /**
+     * @param string $mode
+     */
+    public function setListMode($mode);
+
+    /**
+     * return the list mode.
+     *
+     * @return string
+     */
+    public function getListMode();
+
     /*
      * Configure buttons for an action
      */
-    // public function configureActionButtons(string $action, ?object $object = null): void;
+    // public function configureActionButtons(string $action, ?object $object = null): array;
 
     //TODO: uncomment this method for 4.0
     /*
      * Returns the result link for an object.
+     *
+     * @return string|null
```

Comment:
Since the method already has a return declaration, this docblock seems superfluous.